src/Resources/config/doctrine/model/Look.orm.xml

```diff
@@ -12,6 +12,16 @@
         </id>
 
         <field name="code" column="code" type="string" unique="true"/>
+        <field name="percentageDiscount" column="percentage_discount" type="decimal" precision="10" scale="5"/>
```

Comment:
```suggestion
        <field name="percentageDiscount" column="percentage_discount" type="decimal" precision="5" scale="2"/>
```

A discount can never be more than 100%, so I think this precision and scale is safe

Reply:
Its all about values like `99.999` for example.
See how tax rate is designed at core please.

Reply:
I agree it's nothing, but if you have `precision = 10` and `scale = 5`, you can have a discount like `99999.99999%`. With a scale of `2` you at least constrain the discount to be something like `50.75` and not like `50.75134`